src/jit/codegencommon.cpp

```diff
@@ -11276,6 +11276,26 @@ bool Compiler::IsRegisterPassable(GenTreePtr tree)
 {
     return IsRegisterPassable(gtGetStructHandleIfPresent(tree));
 }
+
+//-----------------------------------------------------------------------------------
+// IsMultiRegReturnedType: Returns true if the type is returned in multiple registers
+// 
+// Arguments:
+//     hClass   -  type handle
+//
+// Return Value:
+//     true if type is returned in multiple registers, false otherwise.
+bool Compiler::IsMultiRegReturnedType(CORINFO_CLASS_HANDLE hClass)
+{
```

Comment:
Multi-reg support is yet to be added to other RyuJIT targets (arm64, x86 and arm32).  Eventually all these routines will be apropriately defined without #if over them.  x64 unix is the first target i am getting multi-reg support added.